.idea/compiler.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="11" />
+    <bytecodeTargetLevel target="17" />
```

Comment:
I'm not sure if this causes any unwanted issues. Did you change this on purpose?

Reply:
No, I didn’t do this on purpose. This sits in the .idea folder. Should changes there be included?

Reply:
I removed the 2 changes in the .idea folder and it still looks like it compiles and runs fine.